econplayground/api/serializers.py

```diff
@@ -237,3 +237,20 @@ class Meta:
         fields = (
             'pk', 'title', 'topic_set',
         )
+
+
+class QuestionSerializer(serializers.ModelSerializer):
```

Comment:
Ideally we won't need a rest API for this feature, as I'm thinking we will implement it in Django rather than React. That could change as we get into it, but for now we don't need these serializers.

Reply:
Alright, I've removed the two serializers and all content dependent on them.